_data/education.yml

```diff
@@ -284,6 +284,16 @@ websites:
       img: udemy.png
       tfa: No
 
+    - name: University of Idaho
+      url: https://www.uidaho.edu
+      img: university-of-idaho_32x32.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      hardware: Yes
+      software: Yes
+      doc: https://support.uidaho.edu/TDClient/Requests/ServiceDet?ID=658
```

Comment:
Perhaps this is a better link for docs? https://support.uidaho.edu/TDClient/KB/ArticleDet?ID=109

Reply:
That also works.